.github/renovate.json5

```diff
@@ -48,7 +50,7 @@
         ".*\\.mk$"
       ],
       "matchStrings": [
-        "\\bghcr\\.io/(?<depName>grafana/grafana-build-tools):(?<currentValue>\\S+)\\b"
+        "ghcr.io/grafana/grafana-build-tools:(?<currentValue>\\S+)"
```

Comment:
I do not like to play loose with regular expressions.

What's the problem with correctly anchoring the regular expression? And what's the problem with correctly escaping things that need escaping?

I almost threw up when I realized that renovate does global matches, because that does make writing regular expressions much more complicated, e.g. you cannot reasonably anchor at the beginning or end of a line.

Reply:
Incidentally, the reason why they do global matches is because they encourage you do to what you are removing here: use named captures to provide configuration values, so I don't quite understand why you are removing that instead of just changing the named capture so that it starts before `ghcr`.

Reply:
> What's the problem with correctly anchoring the regular expression?

This is not a strong opinion.

I've been using renovate and its terrible (and yet awesome) regexes for several years, and I've gone through efforts of making regexes very specific and robust so they don't catch things they shouldn't by mistake. Then I stopped doing it. Since I do the simplest, dumbest, possible regex I've noticed two things:
- Dumb regexes have never caught up things they shouldn't. Not a single time.
- I can actually read the dumb regex three days after I write it.

For those two reasons I keep the shortest, simplest possible one that I cannot anticipate breaking.

> use named captures to provide configuration values

Also not a strong opinion. I've found prematurely generalizing managers also obscures the reasoning for the rule being added, and also adds cognitive complexity (just like the regex).